parent/pom.xml

```diff
@@ -1066,53 +1070,136 @@
 			</properties>
 		</profile>
 
+		<!-- JBIDE-23206
+			 To force a local build of a pull request to use current date/time when generating plugins and features,
+			 instead of using jgit timestamps (which could be in the past, and therefore result in stuff that Eclipse
+			 thinks is OLDER that a current N build, use this "R" profile:
+			 mvn verify -PR
+		-->
 		<profile>
-			<id>test-on-installed-app</id>
```

Comment:
Yeah, I realized later the problem was that you reordered things and git diff was confused.